third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -66,14 +66,16 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 				Computed:      true,
 				ForceNew:      true,
 				ConflictsWith: []string{"name_prefix"},
-				ValidateFunc:  validateGCPName,
+                                ValidateFunc:  validateGCPName,
```

Comment:
I think you converted from tabs to spaces in your changes, do you mind reverting it to keep consistency?

Reply:
I am using VIM on remote linux, not sure where to config tabs to spaces. From that editor, I can't see tabs and spaces. I clicked `enter` from the above line, which leads to this.  This is erb file, I can't run go fmt.